apps/src/sites/hourofcode.com/pages/public/index.js

```diff
@@ -1,20 +1,95 @@
-var google;
-var thanksUrl;
-var signupErrorMessage;
-var censusErrorMessage;
-var hocYear;
+/* globals google thanksUrl signupErrorMessage censusErrorMessage hocYear */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown.jsx';
+import SchoolNotFound from '@cdo/apps/templates/SchoolNotFound.jsx';
+
+var schoolData = {
+  nces: '',
+  schoolName: '',
+  schoolCity: '',
+  schoolState: '',
+  schoolZip: '',
+  schoolType: '',
+  showDropdownError: false,
+  showSchoolNotFoundError: false
+};
+
+function renderSchoolDropdown() {
+  ReactDOM.render (
+    <SchoolAutocompleteDropdown
+      setField={schoolDropdownOnChange}
+      value={schoolData.nces}
+      showErrorMsg={schoolData.showDropdownError}
+    />,
+    $('#school-selector')[0]
+  );
+}
+
+function renderSchoolNotFound() {
+  ReactDOM.render (
+    <SchoolNotFound
+      setField={schoolNotFoundOnChange}
+      schoolName={schoolData.schoolName}
+      schoolType={schoolData.schoolType}
+      schoolCity={schoolData.schoolCity}
+      schoolState={schoolData.schoolState}
+      schoolZip={schoolData.schoolZip}
+      showErrorMsg={schoolData.showSchoolNotFoundError}
+    />,
+    $('#school-not-found')[0]
+  );
+}
```

Comment:
It looks like `school-selector` and `school-not-found` are right next to each other in the DOM. Can we instead render into a single element and have a single root DOM node:

i.e
```
ReactDOM.render(
  <div>
    <SchoolAutocompleteDropdown .../>
    <SchoolNotFound .../>
  </div>,
  document.getElementById('school-autocomplete')
);
```

Reply:
Per Erin's feedback I have now removed the SchoolNotFound component entirely.